src-browserify/core.js

```diff
@@ -0,0 +1,62 @@
+(function (name, context, definition) {
+  if (typeof module != 'undefined' && module.exports) module.exports = definition()
+  else context[name] = definition()
+})('cartodb', this, function() {
+  var cdb = {};
+  cdb.VERSION = "3.15.8";
+  cdb.DEBUG = false;
+  cdb.CARTOCSS_VERSIONS = {
+    '2.0.0': '',
+    '2.1.0': ''
+  };
+  cdb.CARTOCSS_DEFAULT_VERSION = '2.1.1';
+
+  var Loader = require('./core/loader');
+
+  cdb.config = {};
+  cdb.core = {
+    Profiler: require('./core/profiler'),
+    util: require('./core/util'),
+    Loader: Loader
+  };
+  cdb.Image = require('./vis/image.js');
+  cdb.SQL = require('./api/sql');
+  cdb.Tiles = require('./api/tiles');
+
+  cdb.image = {};
+  cdb.geo = {};
+  cdb.geo.ui = {};
+  cdb.geo.geocoder = {};
+  cdb.ui = {};
+  cdb.ui.common = {};
+  cdb.vis = {
+    Loader: Loader
+  };
+  cdb.decorators = {};
```

Comment:
This file was copied from `src/cartodb.js` and adapted to the new bundle. Now, while working on the standard bundle, I realize that most objects added to the cdb/cartodb object here do not makes much sense for the core bundle (e.g. decorators, geo, geo.ui etc.) since they're never loaded/used, So I believe that these could/should be removed, right?